unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -321,11 +337,23 @@ public VoxelEntityHit GetCloserUnselectedVoxelEntityOnPointer()
         return voxelEntityHit;
     }
 
-    public void NewSceneReady(string id)
+    private void NewSceneAdded(ParcelScene newScene)
+    {
+        if (newScene.sceneData.id != sceneToEditId)
+            return;
+
+        Environment.i.world.sceneController.OnNewSceneAdded -= NewSceneAdded;
+
+        FindSceneToEdit();
```

Comment:
Since we already know here the `sceneToEditId`, we could assign directly the scene, `FindSceneToEdit `will choose the nearest scene, but we want the specific one!

Reply:
You're right! Good catch. I have fixed it in https://github.com/decentraland/explorer/pull/2288/commits/2fb2767449b433bd0912255f1eb1c421cd92a4e5. Thanks!